src/applications/static-pages/sass/modules/_m-facility-events.scss

```diff
@@ -0,0 +1,20 @@
+.va-c-event-info {
+  dl {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  dt {
+    width: 33%;
+  }
+  dd {
+    margin-left: auto;
+    width: 66%;
+  }
+
+}
+
+.va-c-event-share {
```

Comment:
Same as above, could we create a class that goes on the `i` elements and style that?

Reply:
Class is on the i element now.